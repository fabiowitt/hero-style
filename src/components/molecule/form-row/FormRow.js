import React from 'react';
import PropTypes from 'prop-types';

import css from './FormRow.module.scss';

const FormRow = ({ children }) => {
  return <div className={css['c-form-row']}>{children}</div>;
};

FormRow.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.array,
    PropTypes.object
  ]).isRequired
};

export default FormRow;
