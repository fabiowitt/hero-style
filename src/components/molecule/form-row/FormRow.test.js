import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import FormRow from './FormRow';

describe('FormRow', () => {
  it('should match with the snapshot', () => {
    const children = 'FormRow';
    const wrapper = shallow(<FormRow>{children}</FormRow>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
