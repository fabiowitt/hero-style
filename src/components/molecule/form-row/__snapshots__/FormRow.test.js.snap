// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormRow should match with the snapshot 1`] = `
<div
  className="c-form-row"
>
  FormRow
</div>
`;
